src/safeds/data/image/containers/_image.py

```diff
@@ -176,3 +176,20 @@ def resize(self, new_width: int, new_height: int) -> Image:
         new_image = Image(data, self._format)
         new_image._image = new_image._image.resize((new_width, new_height))
         return new_image
+
+    def convert_to_grayscale(self) -> Image:
+        """
+        Convert the image to grayscale.
+
+        Returns
+        -------
+        grayscale_image : Image
+            The grayscale image.
+        """
+        data = io.BytesIO()
+        grayscale_image = self._image.convert("L")
+        grayscale_image.save(data, format=self._format.value)
```

Comment:
Wieso wird das Bild gespeichert und nicht einfach zurückgegeben?

Reply:
The `convert_to_grayscale()` method both saves the grayscale image in memory (without needing to convert it again) and returns an `Image` object. I saved the image to ensure that the resulting grayscale image is stored in a format that can be easily accessed and used later if needed.